cou/steps/plan.py

```diff
@@ -57,7 +61,12 @@ async def generate_plan(analysis_result: Analysis) -> UpgradeStep:
         description="Control Plane principal(s) upgrade plan",
         target=target,
         filter_function=lambda app: not isinstance(
-            app, (OpenStackSubordinateApplication, OpenStackAuxiliarySubordinateApplication)
+            app,
+            (
+                OpenStackSubordinateApplication,
+                OpenStackAuxiliarySubordinateApplication,
+                OvnSubordinateApplication,
```

Comment:
is this necessary? `OvnSubordinateApplication` is a subclass of `OpenStackAuxiliarySubordinateApplication`, so `isinstance(OvnSubordinateApplication(), OpenStackAuxiliarySubordinateApplication)` is supposed to return `True`

Reply:
It was not necessary, but because the way that the decorator works, sometimes we need to import and not use it. We are already not using it for the principal, so I agree that we can remove it.